coap/LICENSE

```diff
@@ -0,0 +1,85 @@
+Copyright (c) 2010--2021, Olaf Bergmann and others
```

Comment:
This could be soft link to LICENSE file from libcoap. Please check `expat` component for this.